package.json

```diff
@@ -8,7 +8,9 @@
     "compile:l2": "hardhat starknet-compile",
     "compile": "yarn compile:l1 && yarn compile:l2",
     "deploy:l2:testnet": "hardhat starknet-deploy --starknet-network alpha",
-    "test": "hardhat test",
+    "test:hardhat": "hardhat test",
+    "test:pytest": "py.test",
+    "test": "yarn test:hardhat && yarn test:pytest", 
```

Comment:
Note that we need to install pytest and cairo-lang from pip. Will those packages be installed in this stage?

Reply:
yeah just fixed for the GA now